auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -611,6 +614,7 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   snackbarText: string = null
   showSnackbar = false
   timeoutMs = 4000
+  initialBusinessIdentifierManageBusinessDialog = null
```

Comment:
I think we can use the businessIdentifier?

Reply:
if there are multiple affiliation requests for different business in the same account it will have multiple possible business identifiers. So I am passing this one to open it for specific (this is 2ndary action button on affiliation table)

So I am not 100% sure.

where does this.businessIdentifier come from ?

Reply:
Line 602

Reply:
Ok, yeah, am setting businessIdentifier  too in my event.